CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ ExternalProject_Add(
         "${GOOGLEAPIS_CPP_SOURCE}/google/api/auth.proto"
         "${GOOGLEAPIS_CPP_SOURCE}/google/api/client.proto"
         "${GOOGLEAPIS_CPP_SOURCE}/google/api/resource.proto"
+        "${GOOGLEAPIS_CPP_SOURCE}/google/devtools/cloudtrace/v2/trace.proto"
```

Comment:
Do we also need `tracing.proto`, which has the `TraceService` definition?

Reply:
Yes!  I don't know how I missed that.